coala_quickstart/coala_quickstart.py

```diff
@@ -14,15 +19,37 @@
 
 
 def main():
+    description = """
+coala-quickstart automatically creates a .coafile for use by coala.
+"""
+
+    arg_parser = argparse.ArgumentParser(
+        prog="coala-quickstart",
+        description=description,
+        add_help=True
+    )
+
+    arg_parser.add_argument(
+        '--ci', const=True, action='store_const',
+        help='run in automatic mode without asking any questions.')
+
+    args = arg_parser.parse_args()
+    # Remove custom arg --ci otherwise coalib's arg parser fails
+    if args.ci:
+        sys.argv = [arg for arg in sys.argv if arg != '--ci']
```

Comment:
the coala functions should have an argument so you can pass a custom argv and don't have to modify the global state here